blocks/api/serializer.js

```diff
@@ -139,6 +139,21 @@ export function getBeautifulContent( content ) {
 	} );
 }
 
+export function getBlockContent( block ) {
```

Comment:
Can we add a DocBlock, particularly because it's not obvious how this function is distinct from the other four "get me some content" functions in this file.